OpenSim/Simulation/Model/PhysicalFrame.h

```diff
@@ -140,17 +136,16 @@ class OSIMSIMULATION_API PhysicalFrame : public Frame {
         ScaleFactors */
     void scale(const SimTK::Vec3& aScaleFactors);
 
-    /** @name DEPRECATED API 
+    /** Add a Mesh specified by file name to the list of Geometry owned by the ModelComponent.
+    Scale defaults to 1.0 but can be changed on the call line.
+    */
+    void addMeshGeometry(const std::string &aGeometryFileName, const SimTK::Vec3 scale = SimTK::Vec3(1));
```

Comment:
@aseth1 Good point, it's mainly for convenience as used by our API examples and test cases, but these could change to create the Mesh instead. Why on Frames? because in these cases the user doesn't need to explicitly state the Frame that the Geometry is attached to, so more convenience and less error prone.